README.md

```diff
@@ -88,21 +88,35 @@ There are few different ways to get commuter deployed on your severs:
 
 Firstly, to change default port `4000` set env variables `COMMUTER_PORT`
 
-1. If you want to deploy `master` branch then do `git clone git@github.com:nteract/commuter.git` and run `npm run start:production` on your server
+#### From master branch
+
+1. If you want to deploy from `master`, run:
+
+        git clone git@github.com:nteract/commuter.git
+        cd commuter
+        npm i
+        npm run start:production
+
 1. Currently, the [Demo](https://nteract-commuter.herokuapp.com/) app is deployed on [Heroku](https://www.heroku.com/).
-Instructions for getting started and setting up heroku-cli are [here](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction).
-Once all **required env variables** are set do `git push heroku master`
-1. If you want to deploy npm published packages (**recomended**) then:
-  * Install commuter cli `npm install @nteract/commuter-cli -g`
-  * `exec commuter server` - the service is typically wrapped inside [daemontools](https://cr.yp.to/daemontools.html)
+Heroku provides [a guide for getting started with the heroku CLI](https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction) which will get you
+all set up. And, once all **required env variables** are set run:
```

Comment:
Perhaps:

~~And, once all~~

After all **required env variables are set**, run: